Average.java

```diff
@@ -0,0 +1,31 @@
+package Unit1;
+import java.util.Scanner;
+import java.util.*;
+
+public class Average {
+    public static void main(String[] args) {
+        Cal();
+    }
+
+    public static void Cal() {
+        ArrayList<Integer> nums = new ArrayList<Integer>();

```

Comment:
super fancy! already using ArrayLists...something we will get to in a few months.